README.md

```diff
@@ -107,7 +106,6 @@ You can verify that the started nodes are pushing metrics in the monitoring work
 To control the configured exporters are exposing metrics, connect to a node and execute these `curls` commands :
 - For the Node Exporter : `curl -s http://localhost:9100/metrics` - available on all nodes
 - For the DCGM Exporter : `curl -s http://localhost:9400/metrics` - only available on VM type with NVidia GPU
-- For the Slurm Exporter : `curl -s http://localhost:9080/metrics` - only available on the Slurm scheduler VM
```

Comment:
should we just keep this and add a note explaining that the Slurm exporter will be installed as part of the Slurm project > 4.x.x ?

Reply:
all good for me